### PR TITLE
OCP 4.16 Installation doc updates for IBM Power

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s): 4.16

Issue: https://issues.redhat.com/browse/MULTIARCH-4124

Link to docs preview: https://74756--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/machine-configuration-tasks.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/rhcos-enabling-multipath-day-2.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 [IMPORTANT]
 ====
-Enabling multipathing during installation is supported and recommended for nodes provisioned in {product-title} 4.8 or higher. In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time. For more information about enabling multipathing during installation time, see "Enabling multipathing with kernel arguments on RHCOS" in the _Installing on bare metal_ documentation.
+Enabling multipathing during installation is supported and recommended for nodes provisioned in {product-title} 4.8 or higher. In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time. For more information about enabling multipathing during installation time, see "Enabling multipathing post installation" in the _Installing on bare metal_ documentation.
```
Comment (prb112) [inline modules/rhcos-enabling-multipath-day-2.adoc:13]: 4.8 may want to move to 4.12

**Review by prb112 — APPROVED** [review]
/lgtm

File: post_installation_configuration/machine-configuration-tasks.adoc [hunk 2]
```diff
@@ -57,7 +57,7 @@ include::modules/rhcos-enabling-multipath-day-2.adoc[leveloffset=+2]
 [role="_additional-resources"]
 .Additional resources
 
-* See xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-enabling-multipath_installing-bare-metal[Enabling multipathing with kernel arguments on RHCOS] for more information about enabling multipathing during installation time.
+* See xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-enabling-multipath_installing-bare-metal[Enabling multipathing post installation] for more information about enabling multipathing during installation time.
```
Comment (Srivaralakshmi) [inline post_installation_configuration/machine-configuration-tasks.adoc:60]:
```suggestion
* xref:../installing/installing_bare_metal/installing-bare-metal.adoc#rhcos-enabling-multipath_installing-bare-metal[Enabling multipathing post installation]
```

File: modules/rhcos-enabling-multipath-day-2.adoc (hunk 1 above)
Comment (Srivaralakshmi) [inline modules/rhcos-enabling-multipath-day-2.adoc:13]:
```suggestion
Enabling multipathing during installation is supported and recommended for nodes provisioned in {product-title} 4.8 or later versions. In setups where any I/O to non-optimized paths results in I/O system errors, you must enable multipathing at installation time. For more information about enabling multipathing during installation time, see "Enabling multipathing post installation" in the _Installing on bare metal_ documentation.
```

Comment (Srivaralakshmi) [inline modules/rhcos-enabling-multipath-day-2.adoc:13]: Refer to https://redhat-documentation.github.io/supplementary-style-guide/#admonitions.
As per the SSG, suggest to keep admonitions to a minimum, and avoid placing multiple admonitions close to one another. If multiple admonitions are necessary, restructure the information by moving the less-important statements into the flow of the main content.

Comment (Srivaralakshmi) [inline modules/rhcos-enabling-multipath-day-2.adoc:13]: Suggest moving this to L9 after the H1 heading.

**Review by Srivaralakshmi — COMMENTED** [review]
@alishaIBM Good work, PTAL on my suggestions. Thank you!

File: modules/rhcos-enabling-multipath-day-2.adoc (hunk 1 above)
Comment (alishaIBM) [reply modules/rhcos-enabling-multipath-day-2.adoc]: Thanks @Srivaralakshmi for the comments/suggestions. I have moved above note to line 9.

**Pull request merged**